spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -205,14 +205,18 @@ object DynamicConformanceJob {
     newVersion
   }
 
-  private def initFunctionalExtensions(reportVersion: Int)(implicit spark: SparkSession, dao: MenasDAO, cmd: ConfCmdConfig): Unit = {
+  private def initFunctionalExtensions(reportVersion: Int)(implicit spark: SparkSession,
```

Comment:
I would just make it a standard parameter, same as in **Standardization**. IMHO no need to be `implicit`

Reply:
Now `cmd: ConfCmdConfig` is implicit, but `pathCfg: PathCfg` is not...